packages/core/src/test/shared/sam/sync.test.ts

```diff
@@ -2075,6 +1689,99 @@ describe('ensureBucket', () => {
     })
 })
 
+describe('ensureRepo', () => {
```

Comment:
This is the existing pattern for the current file as well as build and deploy. If we want to implement it here, we should do that for all file for consistency. Yet, this would expand the PR even more.